react/Dropzone.js

```diff
@@ -0,0 +1,3 @@
+import Dropzone from './components/Dropzone/index'
```

Comment:
Could you do this in `typescript`? 🤔

Reply:
Will think about it... But only next week.